projects/js-packages/charts/src/components/leaderboard-chart/stories/index.stories.tsx

```diff
@@ -372,6 +379,22 @@ export const NumberFormatting: Story = {
 	},
 };
 
+export const CustomLabel: Story = {
```

Comment:
```suggestion
export const CustomLabelWithOverlay: Story = {
```

Non-blocking: Since a custom label is also now possible without overlay, is this specifically for the overlay version?

Reply:
There is too much going on here. I split custom label and withOverlay features into two separate stories 6145347 & 6864673.